langchain/agents/agent_toolkits/openapi/planner.py

```diff
@@ -218,12 +218,17 @@ def _create_and_run_api_controller_agent(plan_str: str) -> str:
             for method, route in matches
         ]
         endpoint_docs_by_name = {name: docs for name, _, docs in api_spec.endpoints}
+
         docs_str = ""
```

Comment:
could we factor our docs_str creation into a helper and write one or two simple unit tests for it?